vk_api/audio.py

```diff
@@ -128,18 +128,24 @@ def scrap_data(html):
     soup = BeautifulSoup(html, 'html.parser')
     tracks = []
     for audio in soup.find_all('div', {'class': 'audio_item'}):
-        ai_artist = audio.select('.ai_artist')
-        artist = ai_artist[0].text
-        link = audio.select('.ai_body')[0].input['value']
+        if 'audio_item_disabled' in audio["class"]:
+            # TODO: implement getting data of unavailable track
+            continue
+        else:
```

Comment:
you can delete `else`

Reply:
I don't want to make any more changes under else. I'm gonna just replace lines 132-133 by implemented code in future.

Reply:
`else` block is not needed, until it's not implemented

Reply:
Deal